### ISSUE TITLE
Support for chains that don't have "cosmos" as hrp for bech32 address

### ISSUE THREAD

**Issue by andynog** [informalsystems/ibc-rs#416]
## Crate

modules

## Summary of Bug

Currently, if one of the chains configure on the relayer doesn't have an address starting with "cosmos" , the transactions will fail with a 

```
conn init failed, error:  tx error: GRPC error: GRPC error: status: Unknown, message: "invalid Bech32 prefix; expected akash, got cosmos", details: [], metadata: MetadataMap { headers: {} }
```

I think this might have to do with this code since "cosmos" is hard-coded there https://github.com/informalsystems/ibc-rs/blob/db04e2468adc09551563b09cf137c872ba800df3/modules/src/address.rs#L11

## Steps to Reproduce

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by adizere** [comment]
Closing in favor of #635.

**Comment by romac** [comment]
Fixed in #753
